guides/deployment/releases.md

```diff
@@ -221,7 +236,13 @@ COPY --from=build --chown=nobody:nobody /app/_build/prod/rel/my_app ./
 
 ENV HOME=/app
 
-CMD ["bin/my_app", "start"]
+ENTRYPOINT ["bin/my_app"]
+CMD ["start"]
 ```
 
 At the end, you will have an application in `/app` ready to run as `bin/my_app start`.
+
+A few points about configuring a containerized application:
+
+- If you run your app in a container, the endpoint needs to listen on a "public" IP address like `0.0.0.0` so that the app can be reached from outside the container. Whether the host should publish the container's ports to its own public IP or to localhost depends on your needs.
```

Comment:
If I want to change this, where do I go? What are our defaults? :)

Reply:
It's now specified in `config/runtime.exs` in a new app - https://github.com/phoenixframework/phoenix/blob/master/installer/templates/phx_single/config/runtime.exs#L21
I'm updating the comment to be a little more specific.